package.json

```diff
@@ -41,11 +41,10 @@
     "rimraf": "^6.0.1",
     "svgo": "^3",
     "turbo": "^2.0.11",
-    "typescript": "^5.7.2"
+    "typescript": "^5.8.3"
   },
   "engines": {
-    "node": "^12.22.0 || ^14.17.0 || >=16.0.0",
-    "npm": ">=7"
```

Comment:
You use yarn, this doesn't make a lot sense (given that this is for the workspace, not the packages)
Also updated node support to ESLint-compatible versions.